gnocchi/storage/__init__.py

```diff
@@ -357,6 +357,25 @@ def _store_timeserie_splits(self, metric, keys_and_aggregations_and_splits,
     def _add_measures(self, metric, aggregations, grouped_serie,
```

Comment:
is there a reason you didn't change this method name? too many places to change in test? just asking because this method name is misleading from what it actually does.

Reply:
No, I just did not think about changing it. Let me do that! Do you have a suggestion?